File: БР1.2/Тимаков Егор/labs/src/main/java/org/rentservice/data/response/UserResponse.java
package org.rentservice.data.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserResponse {

        private Long id;

        private String firstName;

        private String lastName;

        private String middleName;

        private String email;

        private String role;

        private Boolean isVerified;

        private LocalDateTime createdAt;
    }

File: БР1.2/Алексеев Андрей/labs/lab1/src/main/kotlin/ru/itmo/pxdkxvan/lab1/user/dto/UserResponse.kt
package ru.itmo.pxdkxvan.lab1.user.dto

import java.time.OffsetDateTime
import java.util.UUID
import ru.itmo.pxdkxvan.lab1.role.dto.RoleResponse

data class UserResponse(
    val id: UUID,
    val roles: List<RoleResponse>,
    val firstName: String,
    val lastName: String,
    val middleName: String?,
    val email: String,
    val phone: String,
    val isVerified: Boolean,
    val createdAt: OffsetDateTime,
    val updatedAt: OffsetDateTime,
)
